README.md

```diff
@@ -1,7 +1,21 @@
-# jargon
+# jargonLint
 
-This project is quite new and a WIP
+The repository to find popular Vale rules that are:
+- **Comprehensive:**  Exhaustively prepared from reliable sources
+- **Easy to Implement:** Tested and well-documented
+- **Permissively Licensed:** MIT compatible licensing
 
-Creation date: 2024-05-27
+## Vale
 
-Please read https://github.com/ccoVeille/jargon/discussions/1
+[Vale](https://vale.sh/) is an open source documentation software, accessible both from the [command-line](https://vale.sh/docs/vale-cli/installation/) or through [integrations](https://vale.sh/docs/integrations/guide/). An important advantage of the software is its capability to support custom [styles](https://vale.sh/docs/topics/styles/) and [lint actions](https://vale.sh/docs/topics/actions/).
+
+Many organizations that use this capability with Vale also publish their configurations with permissive licenses. The repositories with these rulesets can be very useful to learn and adapt from. However, they also reveal that many documentation writers have re-iterated upon similar rules delivering the same results.
```

Comment:
my only remark would be here

it's ok for a first version, but we could mention them in links or in a dedicated list at the end like

```
# Similar vale-package
- vale-at-redhat + link
- suse + link
- ...
```

Reply:
Yes, I agree that we should add links to certain style guides, especially to SUSE and Red Hat. I'm only delaying it for now because I want to add a few things to the repository beforehand.

Reply:
I created:
- #19